cudax/test/memory_resource/any_resource.cu

```diff
@@ -8,27 +8,39 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include <cuda/experimental/memory_resource.cuh>
+#ifndef __CUDA_ARCH__
```

Comment:
Why is this needed?

Reply:
the type-erased memory resources are host-only, no?